_pages/oolleegg55.yaml

```diff
@@ -0,0 +1,5 @@
+---
+githubhandle: oolleegg55
+pageURL: oolleegg55.github.io
```

Comment:
Sorry, @oolleegg55  I don't understand the travis-ci message, either.  I see some error message related to other yaml files, but cannot find yours.

I encounter the same problem too in my pull request and want to figure it out.

@githubteacher Could you teach us how to read the travis-ci message?